specification/src/main/asciidoc/platform/Interoperability.adoc

```diff
@@ -1,8 +1,8 @@
 [[a2845]]
-== Interoperability
+== Application Interoperability
 
-This chapter describes the interoperability
-requirements for the Jakarta™ EE Platform.
+This chapter describes the application interoperability
+features for the Jakarta™ EE Platform.
 
 === Introduction to Interoperability
```

Comment:
Should this title to be updated to `Introduction to Application Interoperability` to sync up with the previous title change?

Reply:
As David pointed out, that proposed title change does have a ripple effect as you discovered.  I'm going to change it back.